### PR TITLE
Feature/admin menu layout

### PR DESCRIPTION
**Pull request by GolfGrab**
Add adminMenuLayout

### CODE REVIEW

File: src/modules/admin/menu/MenuPreviewSection.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { Button, Card } from "antd";
+
+type MenuPreviewSectionProps = {
+  menuLayoutVariant: MenuLayoutVariant;
+};
+
+const MenuPreviewSection: React.FC<MenuPreviewSectionProps> = () => {
+  return (
+    <MenuListCard
+      title={<H3>Menu</H3>}
```
Comment (JiMeow) [inline src/modules/admin/menu/MenuPreviewSection.tsx:15]: change "Menu" -> "All Menu"

File: src/modules/admin/menu/AddOnFormSection.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import styled from "@emotion/styled";
+import { Card } from "antd";
+
+type AddOnFormSectionProps = {
+  menuLayoutVariant: MenuLayoutVariant;
+};
+
+const AddOnFormSection: React.FC<AddOnFormSectionProps> = () => {
+  return (
+    <AddOnFormCard title={<H4>AddOnForm</H4>} bordered={false}>
```
Comment (JiMeow) [inline src/modules/admin/menu/AddOnFormSection.tsx:13]: change "AddOnForm" to "Add-on"

File: src/modules/admin/menu/MenuFormSection.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import styled from "@emotion/styled";
+import { Card } from "antd";
+
+type MenuFormSectionProps = {
+  menuLayoutVariant: MenuLayoutVariant;
+};
+
+const MenuFormSection: React.FC<MenuFormSectionProps> = () => {
+  return (
+    <MenuFormCard title={<H4>MenuForm</H4>} bordered={false}>
```
Comment (JiMeow) [inline src/modules/admin/menu/MenuFormSection.tsx:13]: change "MenuForm" to "Edit"

**Review by JiMeow — APPROVED** [review]
Good work, Can merge with develop and change text later

File: src/modules/admin/menu/MenuPreviewSection.tsx (hunk 1 above)
Comment (GolfGrab) [reply src/modules/admin/menu/MenuPreviewSection.tsx:15]: Thanks will improve this in sub module

File: src/modules/admin/menu/MenuFormSection.tsx (hunk 3 above)
Comment (katisd) [reply src/modules/admin/menu/MenuFormSection.tsx:13]: it's probably both edit and add form.

File: src/modules/admin/menu/AddOnFormSection.tsx (hunk 2 above)
Comment (GolfGrab) [reply src/modules/admin/menu/AddOnFormSection.tsx:13]: Thanks will improve this in sub module

File: src/modules/admin/menu/MenuFormSection.tsx (hunk 3 above)
Comment (GolfGrab) [reply src/modules/admin/menu/MenuFormSection.tsx:13]: Thanks will improve this in sub module
But it can be edit or add

**Review by katisd — APPROVED** [review]
nice!✨

**Pull request merged**